CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.h

```diff
@@ -1,6 +1,12 @@
 #ifndef ECALTPGPARAMBUILDER_H
 #define ECALTPGPARAMBUILDER_H
 
+//Author: Pascal Paganini - LLR
+//Date: 2006/07/10 15:58:06 $
+
+#define CMSSW_VERSION 340
```

Comment:
your PR seems to have added it , so indeed there are likely places in the code with this 6 years obsolete stuff - however we don't need to add new if statements as done in this pr